snap/README.md

```diff
@@ -89,3 +89,31 @@ or
 sudo snap install edgex-app-service-configurable edgex-app-service-configurable_mqtt
 ```
 **Note** – you must ensure that any configuration values that might cause conflict between the multiple instances (e.g. port, log file path, …) must be modified before enabling the snap’s service.
+
+### Secret Store Usage
+Some profile configuration.toml files specify configuration which requires secret-store support, however this snap doesn't fully support secure secrets without manual intervention (see below). The snap can also be configured to use insecure secrets as can be done via docker-compose by setting the option ```security-secret-store=false```. Ex.
+
+```
+sudo snap set edgex-app-service-configurable security-secret-store=false
+```
+
+### Manually install CA root certificate & vault token
+Here's an example of how to use a configuration/profile which includes secrets:
+
+```
+$ sudo snap install edgex-app-service-configurable --channel=latest/candidate/alpha
+$ sudo snap set profile=mqtt
+$ cd /var/snap/edgex-app-service-configurable/current/config/res/mqtt-profile
+$ sudo cp /var/snap/edgexfoundry/secrets/ca/ca.pem .
+$ sudo cp /var/snap/edgexfoundry/secrets/edgex-application-service/secrets-token.json .
+```
+
+Next the profile's configuration.toml file needs to be updated to reference the paths in ```$SNAP_DATA``` of the copied files. The two config fields that need updated are ```  RootCaCertPath``` and ```TokenFile```. The following shows how this can be done via the sed command-line tool, however the file can also be easily updated via your favorite editor.
+
+
+```
+$ sed -i -e 's@/vault/config/pki/EdgeXFoundryCA/EdgeXFoundryCA.pem@/var/snap/edgex-app-service-configurable/current/config/res/mqtt-export/ca.pem@' ./configuration.toml
+$ sed -i -e 's@/vault/config/assets/resp-init.json@/var/snap/edgex-app-service-configurable/current/config/res/mqtt-export/secrets-token.json@' ./configuration.toml
+```
+
+**Note** -- these configuration changes need to be made *before* the service is started for the first time. Otherwise, the changes need to be made directly in Consul, and the service restarted.
```

Comment:
One can also manually remove the the configuration from Consul and restart the service, which I recommend over making the changes in Consul

Reply:
That's true, but this note was about making secure secrets work, not disabling them. The latter is what this PR is all about (i.e. disabling secure secrets via the configure hook) so no Consul changes are required. I could add something about this earlier in the page, but I think it might confuse the average user.

Reply:
Following the steps above, I get journal messages saying the following. Could be my setup:

 http: TLS handshake error from 127.0.0.1:47914: remote error: tls: bad certificate

Reply:
> but this note was about making secure secrets work, not disabling them

My comment is aligned with `making secure secrets work`. Has nothing to do with disabling. Pointing out it is easier to simply remove the configuration and let the service re-seed it, than make the edits in Consul.

Reply:
@lenny-intel I took your suggestion and re-worded this note to recommend deleting the configuration and starting over. Just re-pushed, so marking this resolved.